x/execution/internal/keeper/keeper.go

```diff
@@ -88,6 +88,8 @@ func (k *Keeper) Create(ctx sdk.Context, msg types.MsgCreateExecution) (*executi
 		msg.Request.Tags,
 		msg.Request.ExecutorHash,
 	)
+	exec.BlockHeight = ctx.BlockHeight()
```

Comment:
could be include in the function `executionpb.New(`

Reply:
I like that here it's really something extra that is not really part of the execution, just some kind of metadata